src/btree.c

```diff
@@ -3337,7 +3366,20 @@ btree_print_tree(platform_log_handle *log_handle,
                  btree_config        *cfg,
                  uint64               root_addr)
 {
-   btree_print_subtree(log_handle, cc, cfg, root_addr);
+   btree_print_subtree(log_handle, cc, cfg, root_addr, PAGE_TYPE_BRANCH);
```

Comment:
btree_print_tree should take a page type

Reply:
Done.